test/unit/visitor/kinetic_block.cpp

```diff
@@ -244,6 +244,26 @@ SCENARIO("Convert KINETIC to DERIVATIVE using KineticBlock visitor", "[kinetic][
             REQUIRE(result[0] == reindent_text(output_nmodl_text));
         }
     }
+    GIVEN("KINETIC block with -> reaction statement, indexed COMPARTMENT") {
+        std::string input_nmodl_text = R"(
+            STATE {
+                x[2]
+            }
+            KINETIC states {
+                COMPARTMENT i, vol[i] { x }
+                ~ x[0] + x[1] -> (f(v))
+            })";
+        std::string output_nmodl_text = R"(
+            DERIVATIVE states {
+                x'[0] = ((-1*(f(v)*x[0]*x[1])))/(vol[0])
+                x'[1] = ((-1*(f(v)*x[0]*x[1])))/(vol[1])
+            })";
+        THEN("Convert to equivalent DERIVATIVE block") {
+            auto result = run_kinetic_block_visitor(input_nmodl_text);
+            CAPTURE(input_nmodl_text);
+            REQUIRE(result[0] == reindent_text(output_nmodl_text));
+        }
+    }
```

Comment:
@nrnhines could you please review this test? The two fragments of a MOD file should be equivalent, i.e. the reaction equation is converted to that system of ODEs. The bug was that the term `.../(vol[0])` was missing.

Reply:
looks good to me. It is the same ode form as produced by nocmodl for ```static int _ode_spec1(```